src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -66,6 +82,28 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 		}
 	}
 
+	private _updateSPIDStatus(): void {
+		let activeConnection = TaskUtilities.getCurrentGlobalConnection(this.objectExplorerService, this.connectionManagementService, this.editorService);
+		if (activeConnection) {
+			let uri = this.connectionManagementService.getConnectionUriFromId(activeConnection.id);
+			let info = this.connectionManagementService.getConnectionInfo(uri);
+			if (this.editorService.activeEditor) {
+				// USE ACTIVE EDITOR INFO AS THE SPID WILL BE DIFFERENT FOR EDITOR CONNECTION.
+				let newInfo = this.connectionManagementService.getConnectionInfo(this.editorService.activeEditor.resource.toString());
+				if (newInfo) {
+					info = newInfo;
+				}
+			}
+			let text = '(' + info.spid + ')'
+			let tooltip = 'SPID: ' + info.spid;
```

Comment:
```suggestion
			let tooltip = 'Server Process Id: ' + info.spid;
```

Reply:
If we're spelling it out do we want to localize it?

Reply:
I believe we should. Also localize other strings, 'Server', 'Database', 'Login'